corporate/tests/test_stripe.py

```diff
@@ -714,7 +732,7 @@ def client_billing_post(self, url_suffix: str, info: Mapping[str, Any] = {}) ->
     def client_billing_patch(self, url_suffix: str, info: Mapping[str, Any] = {}) -> Any:
         url = f"/json{self.billing_session.billing_base_url}" + url_suffix
         if self.billing_session.billing_base_url:
-            response = self.client_patch(url, info, subdomain="selfhosting")
+            response = self.client_patch(url, info, subdomain="selfhosting")  # nocoverage
```

Comment:
Does this still not have coverage?

Reply:
Looks like we are using client_patch only for Cloud, since Aman's commits also don't use this. So either we can keep it for consistency or just remove the subdomain block.